File: Ruby-on-Rails/app/controllers/pages_controller.rb
class PagesController < ApplicationController
  layout :choose_layout

  def choose_layout
  action_name.in?(%w[coming_soon maintenance error_404 offline]) ? "base" : "vertical"
  end
  
  def pricing
  end

  def faqs
  end

  def coming_soon
  end

  def maintenance
  end

  def timeline
  end

  def error_404
  end

  def starter
  end

  def offline
  end
end
    

File: ASP.NET-Core-MVC/Tailwick/Controllers/PagesController.cs
using Microsoft.AspNetCore.Mvc;

namespace Tailwick.Controllers
{
    public class PagesController : Controller
    {
        public IActionResult Error404()
        {
            return View();
        }

        public IActionResult Pricing()
        {
            return View();
        }

        public IActionResult Faqs()
        {
            return View();
        }

        public IActionResult ComingSoon()
        {
            return View();
        }

        public IActionResult Maintenance()
        {
            return View();
        }

        public IActionResult Offline()
        {
            return View();
        }

        public IActionResult Timeline()
        {
            return View();
        }

        public IActionResult Starter()
        {
            return View();
        }

    }
}